droid-configs.inc

```diff
@@ -177,6 +177,15 @@ Configure sailfish eg naturally landscape devices like mako
 
 ################################################################
 
+%package flashing
+Group: System
+Summary: On device flashing files for %{rpm_device} hw
```

Comment:
This should provide somekind of generic on-device-flash-tools package or something that package needing the flashing scripts can depend on.